README.md

```diff
@@ -51,3 +51,26 @@ The config passed to Configure can contain the following fields.
 | `maxReconnects`            | Sets the number of NATS server reconnect attempts that will be tried before giving up. If negative, then it will never give up trying to reconnect.                                                                                               | false    | `5`                                |
 | `reconnectWait`            | Sets the time to backoff after attempting a reconnect to a NATS server that the connector was already connected to previously.                                                                                                                    | false    | `5s`                               |
 | `bufferSize`               | A buffer size for consumed messages. It must be set to avoid the [slow consumers](https://docs.nats.io/running-a-nats-service/nats_admin/slow_consumers) problem. Minimum allowed value is `64`                                                   | false    | `1024`                             |
+
+## Destination
+
+### Sending messages
+
+The connector sends message synchronously, one by one.
```

Comment:
Is it possible to send message async?

Reply:
No, it's not with PubSub. The nats.go library locks any writes to a NATS server and unlocks only after getting a result from a server.